### PR TITLE
Silpagadi india

### LINKED ISSUE #16: Add silpagadi India file
- [x] Create a **branch**
- [x] Add file/commits
- [x] Open a PR
- [x] Have a discussion about the PR(COLLABORATE)
- [x] Make edits
- [x] Get :shipit: approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by silpagadi**
closes #16 

hey @garimadhakal does this look like a good list of recommendations?

### CODE REVIEW

File: silpagadi-india.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# India
+### Places to eat
```
Comment (ghost) [inline silpagadi-india.md:3]: yum!

**Comment by ghost** [comment]
Great! 👍

**Comment by silpagadi** [comment]
@garimadhakal 
Added places to go to. Check it out !

**Comment by ghost** [comment]
@silpagadi Nice. How about places to avoid?

**Comment by silpagadi** [comment]
@garimadhakal 
I added the places to avoid. Check it out and let me know :+1:

**Comment by ghost** [comment]
@silpagadi Awesome! 👍 :shipit:

**Pull request merged**